src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -25,9 +25,7 @@
 using Microsoft.PowerFx.Syntax;
 
 namespace Microsoft.PowerFx.Core.Functions
-{
-    using FunctionInfo = Microsoft.PowerFx.Core.Functions.TransportSchemas.FunctionInfo;
```

Comment:
>FunctionInfo [](http://example.com/codeflow?start=10&length=12)

why remove? this looks like it made the code cleaner since we repeat it several times.

Reply:
This is [https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0065](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0065) which isn't enforced but isn't a good idea/good practice.

I've changed it to the line below (out of namespace block) to make it more readable (and with a different name to avoid a conflict)

using FuncInfo = Microsoft.PowerFx.Core.Functions.TransportSchemas.FunctionInfo;